### ISSUE TITLE
Pulumi logos are duplicated

### ISSUE THREAD

**Issue by susanev** [pulumi/docs#11760]
weve got our logos in two different places, lets fix this

https://github.com/pulumi/pulumi-hugo/tree/master/themes/default/static/images/logo
https://github.com/pulumi/pulumi-hugo/tree/master/themes/default/static/logos/brand

**Comment by ericrudder** [comment]
@cnunciato is this one now "fixed" after we re-orged the content, etc?

**Comment by cnunciato** [comment]
No, this is still a thing. Added it to the docs board -- we can pick it up.
